packages/palette/src/elements/Cards/SmallCard.ios.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import { Image, View } from "react-native"
+import { color, space } from "../../helpers"
+import { Flex } from "../Flex"
+import { Spacer } from "../Spacer"
+import { Sans } from "../Typography"
+import { Tag } from "./CardTag.ios"
+import { SmallCardProps } from "./SmallCard.shared"
+
+/**
+ * `SmallCard` is a card with a layout one square image on the left,
+ * one tall or two square images on the right, and text for title and subtitle
+ * at the bottom.
+ */
+export const SmallCard: React.FC<SmallCardProps> = ({
+  images,
+  title,
+  subtitle,
+  tag,
+}) => {
+  return (
+    <View>
+      <View
+        style={{
+          width: "100%",
+          aspectRatio: 1.5 / 1.0,
+          flexDirection: "row",
+          borderRadius: 4,
+          overflow: "hidden",
+        }}
+      >
+        <Flex flex={2} background={color("black10")}>
+          <Image source={{ uri: images[0] }} style={{ flex: 1 }} />
+        </Flex>
+        <Spacer mr={0.3} />
```

Comment:
looks like these separators are supposed to be 2px to me. You can get around styled-system by making it a string `"2px"`

Reply:
For this too, talking with tin we decided to use the closest available `space`, so 0.3 should be alright.

Reply:
actually now that I see the screenshots 0.3 looks too chunky to me and I'd recommend we keep it consistent with the designs, not to mention there's already a bunch of places in the app where we use this visual pattern at 2px so this would create inconsistency.

Reply:
I don't like using random values, but alright, I'll use the 2px.